apps/enc.c

```diff
@@ -51,6 +51,7 @@ typedef enum OPTION_choice {
 const OPTIONS enc_options[] = {
     {"help", OPT_HELP, '-', "Display this summary"},
     {"ciphers", OPT_LIST, '-', "List ciphers"},
+    {"list", OPT_LIST, '-', "List ciphers"},
```

Comment:
Please update `openssl-enc.pod` accordingly. Ideally, you would document `-list` as the main list command and `-ciphers` as an alias for `-list`.